tests/test_fv3run.py

```diff
@@ -454,3 +414,42 @@ def test_get_credentials_args(keyfile, expected_docker_args, expected_bind_mount
 def test_get_local_paths(config_dict, local_paths):
     return_value = fv3config.fv3run._docker._get_local_data_paths(config_dict)
     assert set(return_value) == set(local_paths)  # order does not matter
+
+
+def test_call_via_subprocess_command():
+    import json
+
+    @call_via_subprocess
+    def dummy_function(*obj):
+        pass
+
+    command = dummy_function.command(1, a=1, b=1)
+    args = command.pop()
+    assert json.loads(args) == [[1], dict(a=1, b=1)]
+    assert command == ["python", "-m", dummy_function.__module__]
+
+
+def test_call_via_subprocess_main():
+
+    # need to append outputs here since call_via_subprocess does not
+    # support return arguments
+    output = []
+
+    @call_via_subprocess
+    def dummy_function(*obj):
+        output.append(hash(obj))
+
+    inputs = (1, 2, 3)
+
+    # first call is via "main"
+    serialized_args = dummy_function.command(*inputs)[-1]
+    argv = [None, serialized_args]
+    dummy_function.main(argv)
+    dummy_ans = output.pop()
```

Comment:
If you do add the code that modifies the signature of `command` to be the same as the wrapped function, be sure to test it :)

Reply:
I would rather leave this safety guarantee to another PR unless you think it is very important.

Reply:
It's an easy test. just something like `with pytest.raises(TypeError): command(bad=None, args=True)`. I think it's important enough to include, as it can allow weird, hard-to-track bugs if this regresses in a way that isn't intentional.